models/balancer/ethereum/balancer_ethereum_trades.sql

```diff
@@ -0,0 +1,48 @@
+{{ config(
+    alias = 'trades',
+    partition_by = ['block_date'],
```

Comment:
this can be removed, as it's a view and doesn't store data in the spell itself, rather references other spells